org.eclipse.buildship.ui.test/src/main/groovy/org/eclipse/buildship/ui/view/execution/ExecutionViewExpandAndCollapseAllTest.groovy

```diff
@@ -0,0 +1,47 @@
+package org.eclipse.buildship.ui.view.execution
+
+import com.gradleware.tooling.toolingclient.GradleDistribution
+
+import org.eclipse.debug.core.ILaunchConfiguration
+import org.eclipse.debug.core.ILaunchConfigurationWorkingCopy
+import org.eclipse.debug.core.ILaunchManager
+import org.eclipse.swt.widgets.Tree
+import org.eclipse.swtbot.swt.finder.widgets.SWTBotTree
+import org.eclipse.ui.IWorkbenchPage
+
+import org.eclipse.buildship.core.CorePlugin
+import org.eclipse.buildship.core.launch.GradleRunConfigurationAttributes
+import org.eclipse.buildship.core.util.gradle.GradleDistributionSerializer
+import org.eclipse.buildship.ui.external.viewer.FilteredTree
+import org.eclipse.buildship.ui.test.fixtures.SwtBotSpecification
+import org.eclipse.buildship.ui.util.workbench.WorkbenchUtils
+
+class ExecutionViewExpandAndCollapseAllTest extends BaseExecutionViewTest {
+
+    def "Expand and collapse all"() {
+        setup:
+        def project = sampleProject()
+        importAndWait(project)
+        launchTask('${workspace_loc:/root}', 'foo')
```

Comment:
The `launchTask()` method should take the root project location from a `File` instance.

Reply:
done